src/hotspot/cpu/arm/icache_arm.cpp

```diff
@@ -67,7 +67,7 @@ static int icache_flush(address addr, int lines, int magic) {
 #else
 
 static int icache_flush(address addr, int lines, int magic) {
-  __builtin___clear_cache(addr, addr + (lines << ICache::log2_line_size));
+  __clear_cache((char *)addr, (char *)addr + (lines << ICache::log2_line_size));
```

Comment:
I thought per comments on the previous PR that `__builtin___clear_cache` would work on arm32?

Reply:
yes, sorry